integration-tests/run_cluster.sh

```diff
@@ -1,5 +1,5 @@
 # cleanup 
-for node in druid-historical druid-coordinator druid-overlord druid-router druid-broker druid-middlemanager druid-zookeeper druid-metadata-storage;
+for node in druid-historical druid-coordinator druid-overlord druid-router druid-broker druid-middlemanager druid-zookeeper-kafka druid-metadata-storage;
```

Comment:
Can you add a comment here that `druid-zookeeper-kafka` is zookeeper AND kafka ?